tests/app/main/test_briefs.py

```diff
@@ -938,8 +938,8 @@ def test_existing_nice_to_have_requirements_evidence_prefills_existing_data(self
 
         # Check yesno radio buttons
         for i in (0, 2):
-            assert len(doc.xpath("//*[@id='input-yesNo-" + str(i) + "-yes' and @checked]")) == 1
-        assert len(doc.xpath("//*[@id='input-yesNo-1-no' and @checked]")) == 1
+            assert len(doc.xpath("//*[@id='input-yesNo-" + str(i) + "-1' and @checked]")) == 1
```

Comment:
Yeah, sorry about that. I was merging the radio and boolean question templates. I knew the IDs will change, but haven't realised we might be using them in tests.